config/config.md

```diff
@@ -130,7 +130,9 @@ The following config option are provided by the OpenHIM. All of these options ha
   },
   "reports": {
     // To enable daily and weekly reporting
-    "enableReports": true
+    "enableReports": true,
+    //the default timezone offset
```

Comment:
@bradsawadye 
This Comment is not helpful.
Tell the user what they can do with this field.